test/gtest/ucp/test_ucp_tag.cc

```diff
@@ -601,8 +601,7 @@ class test_ucp_tag_nbx : public test_ucp_tag {
 
         const ucp_ep_config_t *config = ucp_ep_config(sender().ep());
         if (send_dt.buf_length() >= config->tag.rndv.am_thresh.local &&
-            send_dt.buf_length() >= config->am.zcopy_thresh[0] && is_iov() &&
```

Comment:
Please add a comment

Reply:
comment exists:
            /* For zcopy rndv proto v2 + IOV datatype, we send an RTR request,
             * which uses rcache, so counters test is not relevant. */